src/compile/axis/parse.ts

```diff
@@ -317,7 +317,8 @@ function getProperty<K extends keyof VgAxis>(property: K, specifiedAxis: Axis, c
     case 'ticks':
       return properties.ticks('ticks', specifiedAxis, isGridAxis, channel);
     case 'title':
-      return getSpecifiedOrDefaultValue(specifiedAxis.title, properties.title(specifiedAxis.titleMaxLength, fieldDef, model.config));
+      const title = getSpecifiedOrDefaultValue(specifiedAxis.title, properties.title(specifiedAxis.titleMaxLength, fieldDef, model.config));
+      return title === null || title === '' ? undefined : title;
```

Comment:
what if title is `undefined`? 

Why not just 
```
return getSpecifiedOrDefaultValue(specifiedAxis.title, properties.title(specifiedAxis.titleMaxLength, fieldDef, model.config)) || undefined;
```

Reply:
Then we want to use the default, right?

Reply:
I see, but then title is never undefined here I guess. 

Still, I think just do `|| undefined` makes it less confusing.

Reply:
I agree